src/main/java/com/mealmentor/enterprise/service/MealPlanServiceStub.java

```diff
@@ -25,39 +30,75 @@ public MealPlanServiceStub(IMealItemDAO mealItemDAO) {
         this.mealItemDAO = mealItemDAO;
     }
 
+    /**
+     * Fetch meal item by id
+     * @param id a unique identifier for a MealItem
+     * @return MealItem
+     */
     @Override
     public MealItem fetchById(int id) {
         MealItem foundMealItem = mealItemDAO.fetch(id);
         return foundMealItem;
     }
 
+    /**
+     * Delete meal item of specific id
+     * @param id
+     * @throws Exception
+     */
```

Comment:
JavaDoc will add more value if you describe the parameters here, @ param and @ throws.